src/main/java/de/tum/in/www1/artemis/service/ExerciseImportService.java

```diff
@@ -60,11 +60,18 @@ void copyExerciseBasis(final Exercise newExercise, final Exercise importedExerci
                 newExercise.setTeamAssignmentConfig(importedExercise.getTeamAssignmentConfig().copyTeamAssignmentConfig());
             }
         }
-        var plagiarismChecksConfig = new PlagiarismChecksConfig();
-        plagiarismChecksConfig.setSimilarityThreshold(importedExercise.getPlagiarismChecksConfig().getSimilarityThreshold());
-        plagiarismChecksConfig.setMinimumSize(importedExercise.getPlagiarismChecksConfig().getMinimumSize());
-        plagiarismChecksConfig.setMinimumScore(importedExercise.getPlagiarismChecksConfig().getMinimumScore());
-        newExercise.setPlagiarismChecksConfig(plagiarismChecksConfig);
+
+        if (importedExercise.getPlagiarismChecksConfig() != null) {
+            var plagiarismChecksConfig = new PlagiarismChecksConfig();
+            plagiarismChecksConfig.setSimilarityThreshold(importedExercise.getPlagiarismChecksConfig().getSimilarityThreshold());
+            plagiarismChecksConfig.setMinimumSize(importedExercise.getPlagiarismChecksConfig().getMinimumSize());
+            plagiarismChecksConfig.setMinimumScore(importedExercise.getPlagiarismChecksConfig().getMinimumScore());
+            newExercise.setPlagiarismChecksConfig(plagiarismChecksConfig);
```

Comment:
Missing the auto-enabled boolean here. Consider extracting this, so it can be easily extended if another field gets added plus it's available for reuse